assignments/objects.js

```diff
@@ -19,27 +19,83 @@ const example = {
 
 // Write your intern objects here:
 
+const newIntern1 = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const newIntern2 = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
+
+const newIntern3 = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+const newIntern4 = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const newIntern5 = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F"
+}
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(newIntern1.name)
+
 // Kennan's ID
 
+console.log(newIntern2.id)
+
+
 // Keven's email
 
+console.log(newIntern3.email)
+
 // Gannie's name
 
+console.log(newIntern4.name)
+
 // Antonietta's Gender
 
+console.log(newIntern5.gender)
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
+Object.prototype.Hello = function() {
+  return "Hello, my name is Kennan!"
```

Comment:
This adds the `Hello` method to all of the objects. How would you add it to only Kennan's?